File: examples/javascript/get-server-stats.mjs
import { XuiOneClient } from '../../src/js/xui-one-client.js';

const client = new XuiOneClient({
  baseUrl: process.env.XUI_BASE_URL || 'https://PANEL-DOMAIN:9000/ACCESSCODE/',
  apiKey: process.env.XUI_API_KEY || 'YOUR_ADMIN_API_KEY'
});

const serverId = process.env.XUI_SERVER_ID || '1';
console.log(await client.get('get_server_stats', { server_id: serverId }));

File: examples/python/get_server_stats.py
import os
import sys
from pathlib import Path

sys.path.append(str(Path(__file__).resolve().parents[2]))

from src.python.xui_one_client import XuiOneClient

client = XuiOneClient(
    base_url=os.getenv("XUI_BASE_URL", "https://PANEL-DOMAIN:9000/ACCESSCODE/"),
    api_key=os.getenv("XUI_API_KEY", "YOUR_ADMIN_API_KEY"),
    verify_ssl=os.getenv("XUI_VERIFY_SSL", "true").lower() == "true",
)

server_id = os.getenv("XUI_SERVER_ID", "1")
print(client.get("get_server_stats", {"server_id": server_id}))
